src/main/java/cloud/fogbow/ras/core/plugins/interoperability/cloudstack/publicip/v4_9/CloudStackPublicIpPlugin.java

```diff
@@ -145,7 +151,7 @@ PublicIpInstance doGetInstance(@NotNull PublicIpOrder publicIpOrder,
         }
 
         if (asyncRequestInstanceState.isReady()) {
-            return buildReadyPublicIpInstance(asyncRequestInstanceState);
+            return buildReadyPublicIpInstance(asyncRequestInstanceState, cloudStackUser);
```

Comment:
Change the names of all methods that create Instances to create<Something>. In this case, createReadyPublicIpInstance.